code/datums/actions/action.dm

```diff
@@ -1,3 +1,9 @@
+#define VREF_IMAGE_LINKED_OBJ 1
+#define VREF_MUTABLE_ACTION_STATE 2
+#define VREF_MUTABLE_MAPTEXT 3
+#define VREF_MUTABLE_SELECTED_FRAME 4
+// added ontop everything else
+#define VREF_IMAGE_ONTOP 5
```

Comment:
use the defines folder, and comment (ideally with autodoc) what the usecase and purpose for em is please